client/src/components/Overview/StyleSelection.jsx

```diff
@@ -1,42 +1,52 @@
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
 /* eslint-disable import/extensions */
 import React, { useContext } from 'react';
 import ProductContext from '../../context';
 import CartForm from './CartForm.jsx';
 
 export default function StyleSelection({ currentStyle, changeSelectedStyle }) {
   const { styles } = useContext(ProductContext);
-  // use the current style to overlay a checkmark on the thumbnail of that style
-  // when the user clicks a new thumbnail, remove the checkmark from the old one
-  // and overlay it on the new one.
-  // Probably need a local state to keep track of the one that is selected.
-  // Create an onclick handler that takes in the event's value, locates the correct style
-  // from the style list, and passes the object back up the chain. Hopefully...
+
   function handleStyleSelection(event) {
-    event.persist();
     let newSelection;
     const selectionId = event.target.attributes[0].value;
-    for (let i = 0; i < styles.length; i++) {
+    for (let i = 0; i < styles.length; i += 1) {
       if (+(styles[i].style_id) === +(selectionId)) {
         newSelection = styles[i];
```

Comment:
break; after line 16.  But it could be cleaner to use styles.find() (the built-in array method) here.

Reply:
Thanks for pointing that out. I forget about the built-in methods all the time! change made!